contributing_to_docs/doc_guidelines.adoc

```diff
@@ -92,6 +92,36 @@ Try to shorten the file name as much as possible _without_ abbreviating importan
 
 If you create a directory with a multiple-word name, separate each word with an underscore, for example `backup_and_restore`. Do not create a top-level directory in the repository without checking with the docs team. In the main OpenShift docs, you can create one level of subdirectories. In the docs for features that are designed to be used with OpenShift, such as Service Mesh and OpenShift virtualization, you can create two levels of subdirectories.
 
+When creating a new directory or subdirectory, you must create two symbolic links in it:
+
+* An `images` symbolic link to the top-level `images/` directory
+* A `modules` symbolic link to the top-level `modules/` directory
+
+If the directory that contains an assembly does not have the `images` symbolic link, any images in that assembly or its modules will not be included properly when building the docs.
```

Comment:
Did you mean to say 

> If the directory that contains an assembly does not have the `images` **and `modules`** symbolic links ...

Reply:
I didn't, because you technically only need the images link for the image to show up properly.

I'm not 100% positive of the reason the modules link is necessary, but if you want to share it with me, I can include that. The only thing I've seen is that it's helpful when trying to preview assemblies from Atom.

Reply:
@vikram-redhat did you have anything that you wanted me to add here for the reason for including the modules/ symlink?

Reply:
Not sure.... I think I am confused because I am not sure whether you are talking about _the images in the modules in the assemblies_, or the modules themselves, because you are asking folks to have the images symbolic link AND the modules symbolic link, but then giving the rationale for just the images needing the symbolic link (I think).

The modules symbolic link is also needed at all depth levels so that you can just do: 
`include:modules/<module_name>
`
rather than:
`include:../../modules/<module_name>
`

Reply:
@vikram-redhat but unless this recently changed, `ascii_binder` builds from the cwd, so the symlinks are irrelevant for it; I built with symlinks globally deleted locally in just my repo for a year without incident; a build with asciidoctor cares and the build scripts care?

Reply:
@vikram-redhat I'm going to merge this PR in now. If you know of what the rationale is for requiring the modules/ symlink, I'll be happy to update in another PR.